script.js

```diff
@@ -32,14 +32,14 @@ function createAvatar(el) {
 	}
 
 	name = el.getAttribute("data-avatar");
-	if(name) {
-		const idx = name.split('').map(x=> x.charCodeAt(0).toString()).join('') & colors.length
-		el.setAttribute("style", "background-color:" + colors[idx_from_alphabets % colors.length]);
+	if (name) {
+		const idx = name.split('').map(x => x.charCodeAt(0).toString()).join('') % colors.length
+		el.setAttribute("style", "background-color:" + colors[idx % colors.length]);
```

Comment:
i think you dont need '% colors.length' here, you are calculating that in the idx variable above